.github/workflows/go.yml

```diff
@@ -50,6 +50,15 @@ jobs:
           - 1.15.x
           - 1.16.x
           - 1.17.x
+        exclude:
+          - os: macos-latest
+            go-version: 1.12.x
+          - os: macos-latest
+            go-version: 1.13.x
+          - os: macos-latest
+            go-version: 1.14.x
+          - os: macos-latest
+            go-version: 1.15.x
```

Comment:
I believe that we should add 1.16.x here as well, no?

How do we know if it's failing or not?
<img width="856" alt="image" src="https://user-images.githubusercontent.com/50976344/206352208-22356731-6a7e-446b-ba7e-295279ddf1b0.png">

Reply:
agh maybe .. these are just the ones that i saw consistently failing. if you can find a definitive list either from Go or macos that says which Go versions are not supported on macos monterey, then ideally that would be great. But if we keep seeing the 1.16 failing on several PRs without any change that would effect it, then my vote is to also exclude it. @sbiscigl any thoughts here?